deno.json

```diff
@@ -2,9 +2,5 @@
   "tasks": {
     "check": "deno run --allow-read --allow-net https://deno.land/x/link_checker/check.ts",
```

Comment:
Is `--allow-read` to support validating links for pages that are sitting in the file system (as opposed to on the web)?

Reply:
Yeah, we need --allow-read to access local file resource